workshops/templates/workshops/all_trainees.html

```diff
@@ -0,0 +1,154 @@
+{% extends "base_nav_fluid_sidebar.html" %}
+
+{% load crispy_forms_tags %}
+{% load pagination %}
+{% load tags %}
+{% load training_progress %}
+
+{% block content %}
+
+<form role="form" class="form-horizontal training-progress" method="post">
+{% if form.errors.trainees or discard_form.errors.trainees %}
+  <div class="alert alert-danger" role="alert">Select at least one trainee.</div>
+{% elif form.errors or discard_form.errors %}
+  <div class="alert alert-danger" role="alert">Fix errors below and try again.</div>
+{% endif %}
+
+<p>Hint: This list is ordered by last login time by default. You can ask trainees participating in your discussion/demo session to login into AMY and, then, they will appear at the top of this list.</p>
```

Comment:
IMO alerts are for one-off messages, that disappear after reload. This hint is displayed all the time. Are you sure you want to display it as alert?